modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_vbms_uploader.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'pdf_generator_service/pdf_client'
+require 'bd/bd'
+
+module ClaimsApi
+  module V2
+    class DisabilityCompensationVBMSUploader < DisabilityCompensationClaimServiceBase
+      def perform(claim_id) # rubocop:disable Metrics/MethodLength
+        log_job_progress('526 v2 VBMS Uploader job',
+                         claim_id,
+                         'VBMS upload job started')
+
+        claim_object = ClaimsApi::SupportingDocument.find_by(id: claim_id) ||
+                       ClaimsApi::AutoEstablishedClaim.find_by(id: claim_id)
+
+        auto_claim = claim_object.try(:auto_established_claim) || claim_object
+
+        # Reset for a rerun on this
+        set_pending_state_on_claim(auto_claim) unless auto_claim.status == pending_state_value
+
+        uploader = claim_object.uploader
+        uploader.retrieve_from_store!(claim_object.file_data['filename'])
+        file_body = uploader.read
+
+        bd_upload_body(auto_claim:, file_body:)
+
+        log_job_progress('526 v2 VBMS Uploader job',
+                         claim_id,
+                         'Uploaded 526EZ PDF to VBMS')
+
+        log_job_progress('526 v2 VBMS Uploader job',
+                         claim_id,
+                         'VBMS upload succeeded')
+
+        start_claim_establsher_job(auto_claim) if auto_claim.status != errored_state_value
```

Comment:
I would say we move this to look for flashes and queue the flash updater here since that's all the additional submissions we'll be handling in v2. then we could get rid of the call to the claim establisher here as it's not really needed.

Reply:
I moved the flashes into the Docker container job right after setting the claim to 'established' since that was a similar workflow to v1 and seemed to make sense here since I did not want  the upload to BD to interfere with that at all.